cmd/cli/config/util.go

```diff
@@ -0,0 +1,59 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+)
+
+// parseStringSliceToMap parses a slice of strings into a map.
+// Each element in the slice should be a key-value pair in the form "key=value".
+func parseStringSliceToMap(slice []string) (map[string]string, error) {
+	result := make(map[string]string)
+	for _, item := range slice {
+		parts := strings.SplitN(item, "=", 2)
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("expected 'key=value', receieved invalid format for key-value pair: %s", item)
+		}
+		key, value := parts[0], parts[1]
+		result[key] = value
+	}
+	return result, nil
+}
```

Comment:
Already have this in https://github.com/bacalhau-project/bacalhau/blob/bfc436a9afb4b62134321f390a94883130ed39c8/cmd/util/flags/types.go#L172-L180

Reply:
using this now.